pkg/observability/product.ts

```diff
@@ -36,8 +36,6 @@ export function init($plugin: IPlugin, store: any) {
     // @ts-ignore -- though `svg` is not part of the interface, it does work.
     svg:  stsIcon,
     name: OBSERVABILITY_PRODUCT_NAME,
-
-    label:               store.getters['i18n/t']('observability.name'),
```

Comment:
Are we using the `observability.name` from the i18n file somewhere else? Else we can drop it also.

Reply:
yep, we do here `pkg/observability/index.ts`